examples/todos/src/components/Footer.js

```diff
@@ -3,21 +3,14 @@ import FilterLink from '../containers/FilterLink'
 import { VisibilityFilters } from '../actions'
 
 const Footer = () => (
-  <p>
-    Show:
-    {" "}
-    <FilterLink filter={ VisibilityFilters.SHOW_ALL }>
-      All
-    </FilterLink>
-    {", "}
-    <FilterLink filter={ VisibilityFilters.SHOW_ACTIVE }>
-      Active
-    </FilterLink>
-    {", "}
-    <FilterLink filter={ VisibilityFilters.SHOW_COMPLETED }>
-      Completed
-    </FilterLink>
-  </p>
+  <div>
+      <span>Show: </span>
+      {VisibilityFilters.map((filter, idx) => (
+          <FilterLink key={filter.type} filter={filter.type}>
+              {filter.label}
+          </FilterLink>
+      ))}
+  </div>
```

Comment:
I know this is pedantic, but can we leave this alone? I'd rather make the UI and code 1:1, than essentially be doing meta-programming of the UI.

Reply:
Was wondering about this as well. Amended my last commit, should be restored now.